libraries/crypto/src/hmac.rs

```diff
@@ -101,30 +106,6 @@ mod test {
         }
     }
 
-    #[test]
-    fn test_hmac_sha256_empty() {
-        let mut buf = [0; 96];
-        buf[..64].copy_from_slice(&[0x5c; 64]);
-        buf[64..].copy_from_slice(&Sha256::hash(&[0x36; 64]));
-        assert_eq!(hmac_256::<Sha256>(&[], &[]), Sha256::hash(&buf));
-    }
-
-    #[test]
-    fn test_hmac_sha256_examples() {
```

Comment:
Shouldn't we keep those examples with fixed-length key? (i.e. empty message and non-empty message)

Reply:
The empty message case is contained in the tese below, but I added it back with a different key, together with the fox, because everyone likes foxes. :)